outlook/src/classes/Company.ts

```diff
@@ -9,6 +10,12 @@ const ID_COMPANY_FROM_REVEAL: number = -2;
  */
 const ID_COMPANY_EMPTY: number = -1;
 
+export enum EnrichmentStatus {
```

Comment:
Could be great to add some explanations I think about statuses, their use and meaning.

Reply:
Done, explanations added.